src/ros_filter_bias_estimator.cpp

```diff
@@ -117,12 +118,23 @@ void RosFilterBiasEstimator::updateBiasEstimate(Eigen::Vector3d &orientation_mea
                         //  happens when the angle steps over the boundary (e.g. from -(PI-0.0001) to (PI-0.0001)). That is explained and handled in the
                         //  else condition.
                         double offset = FilterUtilities::clampRotation(uncorrected_orientation_estimate_[axis] - orientation_measurement[axis]);
+                        double difference_variance = 0.0;
+                        if(estimate_differences_.size() == num_values_variance_estimation_)
```

Comment:
Do we maintain a style for `robot_localization`? The `(` directly behind the `if` is throwing me off from what I'm used to in `robotic_lawnmower`

Reply:
We do not. It is not our code, so we have not applied our coding style to it. My default coding style is more compact white-space-wise, so I automatically leave out that white space unless the linter corrects it.